flexget/plugins/api_rottentomatoes.py

```diff
@@ -508,7 +509,7 @@ def lists(list_type, list_name, limit=20, page_limit=20, page=None, api_key=None
 
 def movies_search(q, page_limit=None, page=None, api_key=None):
     if isinstance(q, basestring):
-        q = q.replace(' ', '+').encode('utf-8')
+        q = q.replace(' ', '+').encode('ascii', errors='ignore')
```

Comment:
If I search for `Léon` on the web, the query string comes out like `L%E9on`. Based on that, I think the correct line might be `urllib.quote_plus(q.encode('latin1', errors='ignore'))`